android/app/build.gradle

```diff
@@ -83,10 +83,7 @@ project.ext.react = [
     enableHermes: false,
 ]
 
-project.ext.vectoricons = [
-    // TODO: remove Ionicons after updating completely to rnpaper and removing ui-kitten
-    iconFontNames: [ 'Ionicons.ttf', 'MaterialIcons.ttf', 'MaterialCommunityIcons.ttf'] // Name of the font files you want to copy
-]
+project.ext.vectoricons = []
```

Comment:
If I remember correctly if you don't specify the fonts it will copy all the fonts that are bundled in the library